### PR TITLE
[Site 295] Fix ordered and unordered lists

### PR DESCRIPTION
**Pull request by nenispng**
**Link to issue:**
https://labcodes.atlassian.net/browse/SITE-295

**Description:**
Now we have OL and UL styles capable of up to 3 levels nesting
The markers were created using ::before and native Freight Sans font capabilities
Mixins were created so the markers were properly placed and list's width also fit the mother container
New variables to handle lists easily

_Sorry about the reverted commits._

### CODE REVIEW

File: themes/medius/scss/article.scss [hunk 1]
```diff
@@ -81,42 +110,221 @@
...
-    font-size: $fs-topic;
-    line-height: 24px;
+  ol, ul {
+    list-style: none;
+    counter-reset: item;
+    
     font-family: $ff-secondary;
-    position: relative;
-    word-break: break-all;
+    font-feature-settings: $linear-number;
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:146]: It is already `inherit` from `.article-content`

File: themes/medius/scss/mixins.scss [hunk 2]
```diff
@@ -36,3 +36,16 @@
   background-size: contain;
   background-position: center;
 }
+
+@mixin list-item-width($marker-width) {
+  position: relative;
```
Comment (fernandolins) [inline themes/medius/scss/mixins.scss:41]: All li have already this settings.

File: themes/medius/scss/mixins.scss [hunk 3]
```diff
@@ -36,3 +36,16 @@
...
+  width: $marker-width;
+  left: (-$marker-width - $spacing-2);
+  margin-right: -$marker-width;
+}
+
+@mixin fix-list-width($marker-width) {
+  position: relative;
```
Comment (fernandolins) [inline themes/medius/scss/mixins.scss:48]: All li have already this settings.

File: themes/medius/scss/article.scss [hunk 4]
```diff
@@ -81,42 +110,221 @@
...
+
+      &:before {
+        @include list-item-width($ol-lvl-3);
+      }
+    }
+  }
+
+  ul {
+    li {
+      @include fix-list-width($ul-lvl-1);
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:210]: I didn't get it why did you put it here? 
if you have already defined a `ul > li` above

File: themes/medius/scss/article.scss [hunk 5]
```diff
@@ -81,42 +110,221 @@
...
+
+      &:before {
+        @include list-item-width($ul-lvl-1);
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:213]: Same thing here

**Review by fernandolins — CHANGES_REQUESTED** [review]

File: themes/medius/scss/article.scss (hunk 1 above)
Comment (nenispng) [reply themes/medius/scss/article.scss:146]: Fixed

File: themes/medius/scss/mixins.scss (hunk 2 above)
Comment (nenispng) [reply themes/medius/scss/mixins.scss:41]: Fixed

File: themes/medius/scss/mixins.scss (hunk 3 above)
Comment (nenispng) [reply themes/medius/scss/mixins.scss:48]: Fixed

File: themes/medius/scss/article.scss (hunk 4 above)
Comment (nenispng) [reply themes/medius/scss/article.scss:210]: Above it is defined the common settings to OL and UL.
This part is UL specific to fix the LIs widths and make them fit within the container.

File: themes/medius/scss/article.scss (hunk 5 above)
Comment (nenispng) [reply themes/medius/scss/article.scss:213]: Above it is defined the common settings to OL and UL.
This part is UL specific to fix the marker's width settings to make the line break outside the marker area

**Review by fernandolins — APPROVED** [review]

**Pull request merged**